AliakseiLiauchuk/src/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { render } from 'react-dom'
+import { createStore, applyMiddleware } from 'redux'
+import { Provider } from 'react-redux'
+import thunk from 'redux-thunk'
+import { createLogger } from 'redux-logger'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Switch, Route } from 'react-router';
+import { BrowserRouter } from 'react-router-dom';
+import reducer from './reducers'
+import Users from './containers/Users'
+import PageNotFound from './pages/PageNotFound'
+import MainPage from './pages/Main'
+import Layout from './components/Layout'
+
+const middleware = [ thunk ]
+// if (process.env.NODE_ENV !== 'production') {
+//   middleware.push(createLogger())
+// }
+
+const store = createStore(
+  reducer,
```

Comment:
```suggestion
  reducers,
```
так становится понятным, используется ли у вас один reducer или комбинация reducer'ов, мне пришлось зайти в файл 'reducers.js', чтобы догнать логику кода:)